test/testcafe/spec/DeviceProbingChromeDTC_spec.js

```diff
@@ -24,48 +24,47 @@ const mockChromeProbingViewAfterOVProbing = RequestMock()
   .onRequestTo(/\/idp\/idx\/introspect/)
   .respond(identifyWithDeviceProbingLoopback)
   .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
-  .respond((req, res) => {
+  .respond((_, res) => {
     res.statusCode = '200';
     res.headers['content-type'] = 'application/json';
     res.setBody(deviceProbingChromeDTC);
   });
 
 const mockChromeProbingThenIdentify = RequestMock()
   .onRequestTo(/\/idp\/idx\/introspect/)
-  .respond((req, res) => {
+  .respond((_, res) => {
     res.statusCode = '200';
     res.headers['content-type'] = 'application/json';
     res.setBody(deviceProbingChromeDTC);
   })
   .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
-  .respond((req, res) => {
+  .respond((_, res) => {
     res.statusCode = '200';
     res.headers['content-type'] = 'application/json';
     res.setBody(identify);
   });
 
 const mockChromeProbingThenCancel = RequestMock()
   .onRequestTo(/\/idp\/idx\/introspect/)
-  .respond((req, res) => {
+  .respond((_, res) => {
     res.statusCode = '200';
     res.headers['content-type'] = 'application/json';
     res.setBody(deviceProbingChromeDTC);
   })
   .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
-  .respond((req, res) => {
+  .respond((_, res) => {
     res.statusCode = '200';
     res.headers['content-type'] = 'application/json';
     res.setBody(identify);
   })
   .onRequestTo(/\/idp\/idx\/authenticators\/poll\/cancel/)
-  .respond((req, res) => {
+  .respond((_, res) => {
     res.statusCode = '200';
     res.headers['content-type'] = 'application/json';
     res.setBody(identify);
   });
 
-// TODO: OKTA-616189 - implement this view in Gen3
-fixture('Device Challenge Polling View for Chrome DTC').meta('v3', false);
+fixture('Device Challenge Polling View for Chrome DTC').meta('v3', true);
```

Comment:
Sorry to cause you a merge conflict because of this but I just merged the change that means you can remove the `.meta('v3',` part and it will run the test by default.